Makefile

```diff
@@ -62,7 +62,7 @@ aiohttp/%.c: aiohttp/%.pyx $(call to-hash,$(CYS)) aiohttp/_find_header.c
 cythonize: .install-cython $(PYXS:.pyx=.c)
 
 .install-deps: .install-cython $(PYXS:.pyx=.c) $(call to-hash,$(CYS) $(REQS))
-	pip install -r requirements/dev.txt
+	@pip install -r requirements/dev.in -c requirements/dev.txt
```

Comment:
It seems like this may need a more intrusive fix: https://github.com/aio-libs/aiohttp/pull/5470/checks?check_run_id=1879584882#step:7:41